### DISCUSSION TITLE
Freeplane supported my course

### DISCUSSION THREAD

**Discussion by MarCorrado** [freeplane/freeplane#328] [Show and tell]
I was invited to conduct a 2-day professional course on a complex topic. At the beginning I was quite worried about the task. Talking for two days is not an easy task. I knew very well the argument, but I needed to figure out how to traduce my knowledge in a training. I then started Freeplane and began to define the topics that I thought may have been of interest. I defined the broad area and then I started to add branches to each topic with more detailed sub-arguments. I drilled down even more to specific key-points for each argument. I used icons to evidence difficult topics where a test or an exercise was relevant. I was quite surprised to see how fast Freeplane allowed me to define my speech. In a few hours the skeleton of my talk was basically decided. I copied the outline of the map and this allowed me to prepare a short summary for the organizer of the speech. I then moved to Powerpoint and all my slides were easily prepared, just following the map. I had a precise path to follow. Every day I prepared some slides and I used icons in Freeplane to evidence the topics already “processed”. When new ideas arrived, I just had to enter them on the map, in the most relevant position.
Freeplane has really helped me a lot to reach my goal.

**Comment by rexelbartolome** [comment]
that's great! thanks for sharing :)

not sure if you know about the filter system but you can Apply Filter to Selected Node, or Quick Filter. so when your talking about a specific point you can isolate a node and their children so those are the only nodes you see, since you seem have a lot of stuff in your map.  And yea I use Freeplane for teaching all the time, it's so versatile, I can just type up a rough outline and most of the time its good enough to use in a lecture instead of making a powerpoint presentation. it's great :D

**Reply by MarCorrado** [reply]
Thank you. I know the filter system but I never used it much. I must give it a try.
Freeplane is so powerful!
